src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -177,16 +180,16 @@ private List<SubscriptionWithType> filterOtherPartySubscription(OtherParty other
         boolean isSendNewOtherPartyNotification = YesNo.isYes(otherParty.getSendNewOtherPartyNotification());
 
         if (hasAppointee(otherParty.getAppointee(), otherParty.getIsAppointee())
-                && isNotificationEventValidToSendToOtherPartySubscription(otherParty.getOtherPartyAppointeeSubscription(), isSendNewOtherPartyNotification)) {
+                && isNotificationEventValidToSendToOtherPartySubscription(otherParty.getOtherPartyAppointeeSubscription(), isSendNewOtherPartyNotification, "otherPartyAppointee")) {
```

Comment:
Is it worth creatring an enum for PartyMember with the appropriate values here (instead of using literals)? Helps with extensibility too in future. Could stick it in same module/folder as others like;

uk.gov.hmcts.reform.sscs.config.PartyMember.OTHER_PARTY_REP;

etc...